lib/linux_admin/security/modprobe.rb

```diff
@@ -0,0 +1,53 @@
+module LinuxAdmin
+  class Security
+    class Modprobe
+      CONF_FILE_NAME = "/etc/modprobe.d/scap_lockdown"
+      SCAP_MODULES = %w(dccp sctp rds tipc)
+
+      def self.apply_scap_settings(filename = CONF_FILE_NAME)
+        SCAP_MODULES.each do |m|
+          disable_module(m, filename)
+        end
+      end
+
+      def self.disable_module(mod_name, filename)
+        new_line = "install #{mod_name} /bin/true"
+        begin
+          config_text = File.read(filename)
+          new_text = config_text.gsub!(/^install #{mod_name}.*/, new_line)
+        rescue Errno::ENOENT
+          # Okay if file doesn't exist we will create it
+        end
+
+        if new_text
+          File.open(filename, "w") do |file|
+            file.puts(new_text)
+          end
+        else
+          File.open(filename, "a") do |file|
+            file.puts(new_line)
+          end
+        end
```

Comment:
I'm seeing a pattern...this probably should move into a helper method in Common.